source/common/stats/hystrix_stats.cc

```diff
@@ -119,30 +128,31 @@ void HystrixStats::addHystrixCommand(std::stringstream& ss, std::string cluster_
   addIntToStream("rollingCountBadRequests",         0,          cluster_info);
   addIntToStream("currentConcurrentExecutionCount", 0,          cluster_info);
   addIntToStream("latencyExecute_mean",             0,          cluster_info);
+
+  // latency information can be  taken rom hystogram, which is only available to sinks
```

Comment:
A comment for the whole function. We should consider creating a const variable for all constant values here. Instead of passing them "hard coded"

Reply:
ok. something like "NA"?  better idea for a name?